core/templates/dev/head/editor/exploration_history.html

```diff
@@ -20,12 +20,10 @@
   <md-card class="oppia-editor-card">
     <h3>List of Changes</h3>
     <div ng-repeat="versionCheckbox in versionCheckboxArray " class="row">
-      <div class="col-sm-2 col-md-2 col-lg-2">
+      <div class="col-sm-2 col-md-4 col-lg-4">
         <input type="checkbox" name="compareVer" ng-model="versionCheckbox.selected" ng-init="init()" ng-click="changeSelectedVersions($event, versionCheckbox.vnum)" ng-disabled="isCheckboxDisabled(versionCheckbox.vnum)" ng-hide="comparisonsAreDisabled" class='protractor-test-history-checkbox-selector'>
-        [<[versionCheckbox.vnum]>]
-      </div>
-      <div class="col-sm-2 col-md-2 col-lg-2">
-        <strong><profile-link-text username="explorationVersionMetadata[versionCheckbox.vnum].committerId"></profile-link-text></strong>
+         [<[versionCheckbox.vnum]>]
+         <strong><profile-link-text username="explorationVersionMetadata[versionCheckbox.vnum].committerId"></profile-link-text></strong>
```

Comment:
It's not required to push again if your build fails. If all tests passes on your local machine and you think the Travis build failed due to some glitch, you can ask us for a restart.